finish/src/test/java/it/io/openliberty/guides/todolistSample/TodoIT.java

```diff
@@ -12,16 +12,16 @@
  // end::copyright[]
 package it.io.openliberty.guides.todolistSample;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
 import org.apache.http.HttpResponse;
 import org.apache.http.client.HttpClient;
 import org.apache.http.client.methods.CloseableHttpResponse;
 import org.apache.http.client.methods.HttpGet;
 import org.apache.http.impl.client.HttpClients;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
-public class TodoTest {
+public class TodoIT {
```

Comment:
May be good to have the test execution order here as sample. Just so that if we need to use it, we don't need to go dig it up elsewhere.